### PR TITLE
Preparing release 1.17.0

### LINKED ISSUE #2057: Release 1.17.0
The tracking issue for release 1.17.0.

@jaegertracing/jaeger-maintainers please add PRs/issues which should go to 1.17 into milestone https://github.com/jaegertracing/jaeger/milestone/9.

Here is diff `1.16-master` https://github.com/jaegertracing/jaeger/compare/v1.16.0...master

@everett980 @rubenvp8510 could you please release jaeger-ui 1.7.0?

### PR DESCRIPTION
**Pull request by albertteoh**
Resolves #2057

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,53 @@ Changes by Version
...
+* Refactor collector mains ([#2060](https://github.com/jaegertracing/jaeger/pull/2060), [@jpkrohling](https://github.com/jpkrohling))
+* Clarify that "kafka" is not a real storage backend ([#2066](https://github.com/jaegertracing/jaeger/pull/2066), [@yurishkuro](https://github.com/yurishkuro))
+* Added -trimpath option to go build ([#2064](https://github.com/jaegertracing/jaeger/pull/2064), [@kadern0](https://github.com/kadern0))
+* Use memory size flag to activate dyn queue size feature ([#2059](https://github.com/jaegertracing/jaeger/pull/2059), [@jpkrohling](https://github.com/jpkrohling))
+* Add before you push to the queue to prevent race condition on size ([#2044](https://github.com/jaegertracing/jaeger/pull/2044), [@joe-elliott](https://github.com/joe-elliott))
+* Count received batches from conforming clients ([#2030](https://github.com/jaegertracing/jaeger/pull/2030), [@yurishkuro](https://github.com/yurishkuro))
+* Agent: Do not increment data loss counters on the first client batch ([#2028](https://github.com/jaegertracing/jaeger/pull/2028), [@yurishkuro](https://github.com/yurishkuro))
```
Comment (albertteoh) [inline CHANGELOG.md]:
```suggestion
* [agent] Do not increment data loss counters on the first client batch ([#2028](https://github.com/jaegertracing/jaeger/pull/2028), [@yurishkuro](https://github.com/yurishkuro))
```

**Comment by pavolloffay** [comment]
@albertteoh could you please sign DCO?

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,53 @@ Changes by Version
...
+
+#### Breaking Changes
+
+#### New Features
+
+* [tracegen] Add service name as a command line option ([#2080](https://github.com/jaegertracing/jaeger/pull/2080), [@kevinearls](https://github.com/kevinearls))
+* Allow the Configuration of Additional Headers on the Jaeger Query HTTP API ([#2056](https://github.com/jaegertracing/jaeger/pull/2056), [@joe-elliott](https://github.com/joe-elliott))
+* Warn about time adjustment in tags ([#2052](https://github.com/jaegertracing/jaeger/pull/2052), [@bobrik](https://github.com/bobrik))
+* Add CLI flags for Kafka batching params ([#2047](https://github.com/jaegertracing/jaeger/pull/2047), [@apm-opentt](https://github.com/apm-opentt))
+* Add common TLS configuration ([#1838](https://github.com/jaegertracing/jaeger/pull/1838), [@backjo](https://github.com/backjo))
```
Comment (pavolloffay) [inline CHANGELOG.md]: this is not new feature. It was refactoring PR

Comment (albertteoh) [reply CHANGELOG.md]: Okay, I've moved it to Minor improvements.

**Comment by albertteoh** [comment]
> @albertteoh could you please sign DCO?

The suggested DCO resolution `git rebase HEAD~69 --signoff` messed by my local (and remote) repo :(. 

I'm going to redo this release again, taking into account all your feedback. Sorry for the confusion.

**Comment by jpkrohling** [comment]
> messed by my local (and remote) repo :(

For reference, it might be easier to work on topic branches, keeping your fork's `master` (local and remote) in sync with the upstream's `master` (this repo). Then you could do a `git rebase master --signoff` from your topic branch. Changes to the upstream master wouldn't mess your rebase.

**Comment by albertteoh** [comment]
> For reference, it might be easier to work on topic branches, keeping your fork's `master` (local and remote) in sync with the upstream's `master` (this repo). Then you could do a `git rebase master --signoff` from your topic branch. Changes to the upstream master wouldn't mess your rebase.

Thanks @jpkrohling! I'll definitely give your suggested workflow a try.

**Pull request closed without merging**